.ci/pipeline/ci.yml

```diff
@@ -83,6 +84,43 @@ jobs:
     condition: failed()
     continueOnError: true
 
+- job: 'LinuxMakeGNU_OpenBLAS_ARM'
```

Comment:
i thin that original LinuxMakeGNU have to be renamed to _x86

Reply:
Or might be even different question - do we even need OpenBlAS x86 build to be validated?

arm openBLAS build should be able to validate most of the things?

Reply:
In my opinion we can keep both OpenBLAS x86 and OpenBLAS arm in this PR, since examples validation is yet not covered for OpenBLAS arm. Once we have examples validation supported with OpenBLAS Arm build then we can take decision to keep or remove  openBLAS x86.

Reply:
Sounds good

Reply:
I would still recommend to keep both even in future since arm side of things might deviate.

Reply:
yes, but there is no users for BLAS backend on x86 yet.   And at the end we have to fit in to reasonable CI. 
One of option would be to run blas_x86 configuration in nightly testing for example.